corehq/apps/app_manager/xpath.py

```diff
@@ -29,7 +30,7 @@ def dot_interpolate(string, replacement):
     return re.sub(DOT_INTERPOLATE_PATTERN, repl, string)
 
 
-def interpolate_xpath(string, case_xpath=None):
+def interpolate_xpath(string, case_xpath=None, module=None):
```

Comment:
I don't think you can get away without adding more args unless we change the way the function works. But I do think that it would be better to pass in an arg like `fixture_xpath` which makes it clear what its doing.